ipaserver/install/server/replicainstall.py

```diff
@@ -836,14 +836,12 @@ def install_check(installer):
             root_logger.debug('No IPA DNS servers, '
                               'skipping forward/reverse resolution check')
 
-        kra_enabled = remote_api.Command.kra_is_enabled()['result']
```

Comment:
If I understand this correctly, before this patch, KRA would be installed automatically if it was present on the master server. With this patch, `--setup-kra` would have to be explicitly passed to `ipa-replica-install`.

Is this the case or am I missing something? Is this behaviour change intended?